product_docs/docs/biganimal/release/using_cluster/05a_deleting_your_cluster.mdx

```diff
@@ -27,3 +27,7 @@ You can restore your deleted cluster for as long as the backup is available.
 3.  On the Restore Cluster page, review your selections in the **Cluster Summary**. Select **Restore Cluster** to begin the restore process.
 
     When the process completes, the restored cluster is available on the [Clusters](https://portal.biganimal.com/clusters) page.
+
+!!!note
+To restore a TDE enabled cluster, the TDE key must match with source cluster encryption key. In case a different key is used the restore operation fails.
```

Comment:
Another point we should mentioned is: If source cluster is not encrypted and replica/restore cluster should not allow to enable the TDE as source cluster is not enabled with TDE. 

Replica/restore work through physical backups and WAL files so when user try to restore cluster with TDE enabled having source cluster is non-TDE, postgres will try to decrypt something where it is not require any decryption...

@GabriFedi97 correct me in case of anything wrong.